code/modules/paperwork/stamps.dm

```diff
@@ -70,11 +89,47 @@
 	icon_state = "stamp-clown"
 	_color = "clown"
 
+/obj/item/weapon/stamp/clown/try_stamp(mob/user,obj/item/weapon/paper/P)
+	var/clown = FALSE
+	if(clumsy_check(user))
+		clown = TRUE
+	if(isrobot(user))
```

Comment:
note to self: this check is redundant, clumsy_check already accounts for borgs with the clown quirk